### ISSUE TITLE
[bug] rotated video

### ISSUE THREAD

**Issue by ignacio82** [jlengrand/FaceMovie#10]
I was playing with Facemovie today and notice a bug. When i make a movie with 4 pictures everything works fine. When I added a 5th i get the weird 45 degrees rotated movie.
Also, facemovie is finding more than 1 face in photos with only 1 face. Is there an easy way to train it?

Best,

-Ignacio

**Comment by jlengrand** [comment]
Hi Ignacio, 

This is not surprising that more than one face is found (For some samples I can raise up to 4). 
But no worry, each face comes with a confidence level, and only the most probable candidate is used later on.

I should change the log message though, to avoid confusing users.

Concerning the rotated movie, I never experienced that before :s, could you provide me with your pictures and the video that you get so that I can investigate that? 

BTW, I merged your branch back into master yesterday night. You might want to retrieve the last version of the code : 

```
git clone git://github.com/jlengrand/FaceMovie.git 
```

in a clean repository. 

Thanks for your support

**Comment by ignacio82** [comment]
I'll send you the pictures in a private message.

Thanks!

On Wed, Jun 13, 2012 at 3:58 PM, julien Lengrand-Lambert <
reply@reply.github.com

**Comment by jlengrand** [comment]
Could you please provide me the output from this command in a terminal ? 

```
sudo dpkg --get-selections | grep open
```

**Comment by ignacio82** [comment]
$ sudo dpkg --get-selections | grep open
fonts-opensymbol install
libfile-desktopentry-perl install
libopenal-data install
libopenal1 install
libopenal1:i386 install
libopencc1 install
libopencore-amrnb-dev install
libopencore-amrnb0 install
libopencore-amrwb-dev install
libopencore-amrwb0 install
libopencv-calib3d2.3 install
libopencv-contrib2.3 install
libopencv-core2.3 install
libopencv-features2d2.3 install
libopencv-flann2.3 install
libopencv-gpu2.3 install
libopencv-highgui2.3 install
libopencv-imgproc2.3 install
libopencv-legacy2.3 install
libopencv-ml2.3 install
libopencv-objdetect2.3 install
libopencv-video2.3 install
libopenexr6 install
libopenobex1 install
libqt4-opengl install
libqt4-opengl:i386 install
nautilus-open-terminal install
netcat-openbsd install
openjdk-6-jre install
openjdk-6-jre-headless install
openjdk-6-jre-lib install
openoffice.org-hyphenation install
openprinting-ppds install
openssh-client install
openssh-server install
openssl install
python-opencv install
python-opengl install
python-openssl install
python-pyopencl install
ttf-opensymbol install
virtuoso-opensource-6.1-bin install
virtuoso-opensource-6.1-common install
vmware-view-open-client install
xserver-xorg-video-openchrome install

On Wed, Jun 13, 2012 at 4:29 PM, julien Lengrand-Lambert <
reply@reply.github.com

**Comment by jlengrand** [comment]
Thanks ! 
I was able to reproduce the bug. 
It seems like it comes from the OpenCV version packaged in Ubuntu 12.04 

```
libopencv-calib3d2.3 install
libopencv-contrib2.3 install
libopencv-core2.3 install
libopencv-features2d2.3 install
libopencv-flann2.3 install
libopencv-gpu2.3 install
libopencv-highgui2.3 install
libopencv-imgproc2.3 install
libopencv-legacy2.3 install
libopencv-ml2.3 install
libopencv-objdetect2.3 install
libopencv-video2.3 install
```

Let me some time to find a workaround now ;)

**Comment by jlengrand** [comment]
I could reproduce the bug by installing the Ubuntu OpenCV packages. 

It seems like the problem comes from the video plugin of the 2.3 release. 
For some reason, some output video sizes are not well processed. 

I found a quick fix (resizing the output image in setting the max horizontal size), but this needs more attention. 

Thanks for reporting the problem !

I keep the issue open until I find a better solution

**Comment by jlengrand** [comment]
Turns out I was wrong. 

It appears that the bug was a bit more trickier than that. I found out that the problem was coming from the size of the output video.
More information can be found on [Stack Overflow](http://stackoverflow.com/questions/11440502/weird-behaviour-using-writeframe-in-opencv).

Bug was solved in commit b503f122f64d3a6c67f3d373e6be548421c20e0b
